bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3877,6 +3848,47 @@ void ReplicaImp::executeRequestsInPrePrepareMsg(concordUtils::SpanWrapper &paren
   }
 }
 
+void ReplicaImp::executeRequestsAndSendResponses(IRequestsHandler::ExecutionRequestsQueue &accumulatedRequests,
+                                                 const std::string &batchCid,
+                                                 concordUtils::SpanWrapper &span) {
+  if (ReplicaConfig::instance().blockAccumulation) {
+    LOG_DEBUG(GL, "Executing all the requests of preprepare message with cid: " << batchCid << " With accumulation");
+    {
```

Comment:
Nit: move "{" to the end of the upper line

Reply:
Clang fix it at moving it one line down.